$NetBSD: patch-Makefile,v 1.1 2014/11/07 19:19:24 steven43126 Exp $

Adds DESTDIR support to makefile, pull request for upstream
created https://github.com/postmodern/chruby/pull/294

--- Makefile.orig	2013-12-04 21:53:30.000000000 +0000
+++ Makefile
@@ -53,10 +53,10 @@ tag:
 release: tag download sign
 
 install:
-	for dir in $(INSTALL_DIRS); do mkdir -p $(PREFIX)/$$dir; done
-	for file in $(INSTALL_FILES); do cp $$file $(PREFIX)/$$file; done
-	mkdir -p $(DOC_DIR)
-	cp -r $(DOC_FILES) $(DOC_DIR)/
+	for dir in $(INSTALL_DIRS); do mkdir -p $(DESTDIR)/$(PREFIX)/$$dir; done
+	for file in $(INSTALL_FILES); do cp $$file $(DESTDIR)/$(PREFIX)/$$file; done
+	mkdir -p $(DESTDIR)/$(DOC_DIR)
+	cp -r $(DOC_FILES) $(DESTDIR)/$(DOC_DIR)/
 
 uninstall:
 	for file in $(INSTALL_FILES); do rm -f $(PREFIX)/$$file; done
